src/pj_factors.c

```diff
@@ -72,7 +72,7 @@ int pj_factors(LP lp, PJ *P, double h, struct FACTORS *fac) {
 
         /* convergence */
         if (!(fac->code & IS_ANAL_CONV)) {
-            fac->conv = - atan2(fac->der.y_l, fac->der.x_l);
+            fac->conv = - atan2(fac->der.x_p, fac->der.y_p);
```

Comment:
I write it myself with a help of colleague who is a mathematician. You can include it in documentation, but it can be shortened or expanded depending on the purpose of documentation. The definition of meridian convergence is taken from [https://www.ngs.noaa.gov/CORS/Articles/SolerFuryASCE.pdf] , which is again taken from Geodetic Glossary by National Geodetic Survey (U.S.). The derivation of equations is an application of mathematical analysis, and it can be written in even simpler form.

The convention is the same (positive when meridian is west of grid north) , and calculations are same for conformal projections because the partial derivatives are also corrected to all have same sign convention which was not the case before, and was a reason for extra minus in previous formula.